Dockerfile_build

```diff
@@ -16,8 +16,11 @@ RUN apt-get update && \
     rm -rf /var/lib/apt/lists/*
 
 # Install ragel
-ENV RAGEL_VERSION=6.10
-RUN curl http://www.colm.net/files/ragel/ragel-${RAGEL_VERSION}.tar.gz -O && \
+ENV RAGEL_VERSION=7
```

Comment:
@adrian-thurston do I need to specify exactly which version of ragel we wanna build with here? I'm also not sure if there are any other spots where I need to update the ragel version. I looked at the CI config file and it didnt specify a version of ragel there.